File: Smart_Accounting/Smart_Accounting.Domain/Supplier/Suppliers.cs
/*
 * @CreateTime: Oct 8, 2018 3:41 PM
 * @Author:  Mikael Araya
 * @Contact: MikaelAraya12@gmail.com
 * @Last Modified By:  Mikael Araya
 * @Last Modified Time: Nov 2, 2018 3:06 PM
 * @Description: Modify Here, Please 
 */
using System;
using System.Collections.Generic;
using Smart_Accounting.Domain.AccountCharts;

namespace Smart_Accounting.Domain.Supplier {
    public class Suppliers {
        public Suppliers()
        {
            SupplierAccount = new HashSet<SupplierAccount>();
        }

        public uint Id { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
        public string PhoneNo { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string SubCity { get; set; }
        public string HouseNo { get; set; }
        public string PostalCode { get; set; }
        public float Balance { get; set; }
        public sbyte Active { get; set; }
        public DateTime? DateCreated { get; set; }
        public DateTime? DateUpdated { get; set; }

        public ICollection<SupplierAccount> SupplierAccount { get; set; }
    }
}

File: Frontend/src/app/modules/suppliers/suppliers.ts
export class Suppliers {
    id: number;
    FullName: string;
    Email: string;
    PhoneNo: string;
    Country: string;
    City: string;
    SubCity: string;
    HouseNo: string;
    PostalCode: string;
    Balance: number;
    Active: number;
    BankAccounts: SupplierAccount[] = [];


    constructor(values: Object = {}) {
        Object.assign(this, values);
      }
}
export class SupplierAccount {
    AccountNumber: string;
    BankName: string;
}
